src/backend/distributed/shared_library_init.c

```diff
@@ -1575,6 +1654,19 @@ RegisterCitusConfigVariables(void)
 		GUC_UNIT_KB | GUC_STANDARD,
 		NULL, NULL, NULL);
 
+	DefineCustomIntVariable(
+		"citus.max_logical_replication_senders_with_cpu_priority_changed",
```

Comment:
this seems like jibberish to me :)

Reply:
I'm wondering how important it is that this be configurable. We could consider using half the CPUs.